nixos/doc/manual/default.nix

```diff
@@ -265,6 +265,7 @@ in rec {
       xsltproc \
         ${manualXsltprocOptions} \
         --stringparam target.database.document "${olinkDB}/olinkdb.xml" \
+        --stringparam id.warnings "1" \
```

Comment:
You're my hero for this. Maybe better to extend this list: https://github.com/NixOS/nixpkgs/blob/4c0230eb566d48d7021f567897ee6d630c35a04a/nixos/doc/manual/default.nix#L132 ?

Reply:
Well, this seems to only impact HTML documents. But adding it to the global options may be better if someone only tests it with some other version without generating the HTML. Not too sure really. What do you think ?